setup.py

```diff
@@ -56,6 +56,8 @@
     'click>=7.0,<8.0',
     'jsonschema[format]>=2.6.0,<2.7',
     'kombu>=4.2.0,<5.0',
+    'kubernetes>=6.0.0',
```

Comment:
Since `reana-client` also depends on `reana-commons`, this would bring K8s to the user space world as well...
What about having an option `pip install reana-commons[kubernetes]` so that the usual `reana-client` world wouldn't need this and only `reana-cluster` world would?